File: challenges/1-media-alunos/ruby/alessandro2702/media.rb
def CalculoMedia(n1,n2)
    result = (n1+n2)/2.to_f
    result.round(2)
end

puts "Digite a primeira nota"
a = gets.chomp.to_i
puts "Digite a segunda nota"
b = gets.chomp.to_i

media = CalculoMedia(a,b)

puts "a média é #{media}"

File: challenges/1-media-alunos/elixir/cristuker/media.ex
IO.puts "Calculador de média"
n1 = IO.gets "Digite a primeira nota\n-> "
n2 = IO.gets "Digite a segunda nota \n-> "
{n1Int,_} = :string.to_integer(n1)
{n2Int,_} = :string.to_integer(n2)
total = n1Int+n2Int
media = div(total, 2)

IO.puts media
